### DISCUSSION TITLE
Every blog is now duplicate!

### DISCUSSION THREAD

**Discussion by ObfuscationJones** [TumblThreeApp/TumblThree#355] [Q&A (Question & Answer)]
**Describe the bug**
Since yesterday, all blogs in TumblThree (latest built) are duplicate.
- if I add a new blog, it will appear once. After restarting the application, it appears also twice.
- if I remember one of the two entries for a blog, it will appear only once until I restart the application ; it will then have entirely disappeared.
- when the list is not ordered, the duplicates appear after the "normal" ones (so from aaaaa.tumblr.com to zzzz.tumblr.com, and then it restarts, from aaaaa.tumblr.com to zzzz.tumblr.com)

The index files haven't been changed, so it really seems to be a display bug. But it's annoying because as soon as you sort the blogs list and want to queue them, you either have each blog "downloaded" twice, or have to click on every other entry.

**To Reproduce**
I have no idea because it came out of nowhere. I opened the application yesterday, sorted by "last added" and realised every single blog was listed twice.

**Desktop (please complete the following information):**
 - TumblThree version: 2.8.2
 - OS: Windows 10 Home
 - Browser Firefox 105.01

**Additional context**

**Comment by thomas694** [comment]
I assume you haven't updated the app yesterday. Anyway, the latest version doesn't contain any change that could lead to this.

Reading your description I make a guess. Have you tested and changed settings yesterday, especially `Collections` in the `Advanced` tab? It looks like you have two collections pointing to the same download location. That should also be visible by different texts in the `Collection` column which you can enable in the blog list header.

**Comment by ObfuscationJones** [comment]
Excellent guess. I didn't realise the download was so critical to Collections (but now I indeed understand why). I removed the said Collection (that I wasn't using yet) and it indeed fixed the issue!
Thanks!
